fearless/Modules/Wallet/TransferConfirm/TransferConfirmViewModelFactory.swift

```diff
@@ -105,6 +105,7 @@ final class TransferConfirmViewModelFactory {
             icon: assetId.icon,
             symbol: asset.symbol,
             balance: displayBalance,
+            fiatBalance: nil,
```

Comment:
why nil there ?

Reply:
because this is unused view model factory